include/aos/common/tools/string.hpp

```diff
@@ -163,6 +163,22 @@ class String : public Array<char> {
         return Resize(from - begin());
     }
 
+    /**
+     * Removes trailing characters.
+     *
+     * @param chars characters to be removed.
+     */
+    void RTrim(const String& chars) {
```

Comment:
Rename to `RightTrim`, add `LeftTrim` and `Trim` (both).

Reply:
fixed